content/zh/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -1,65 +1,54 @@
 ---
-title: 已完成资源的 TTL 控制器
+title: 已完成作业的自动清理
 content_type: concept
 weight: 70
 ---
 <!--
-title: TTL Controller for Finished Resources
+title: Automatic Clean-up for Finished Jobs
 content_type: concept
 weight: 70
 -->
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.21" state="beta" >}}
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
 
 <!--
-The TTL controller provides a TTL mechanism to limit the lifetime of resource
-objects that have finished execution. TTL controller only handles
-{{< glossary_tooltip text="Jobs" term_id="job" >}} for now,
-and may be expanded to handle other resources that will finish execution,
-such as Pods and custom resources.
+TTL-after-finished {{<glossary_tooltip text="controller" term_id="controller">}} provides a 
+TTL (time to live) mechanism to limit the lifetime of resource objects that 
+have finished execution. TTL controller only handles 
+{{< glossary_tooltip text="Jobs" term_id="job" >}}.
 -->
-TTL 控制器提供了一种 TTL 机制来限制已完成执行的资源对象的生命周期。
-TTL 控制器目前只处理 {{< glossary_tooltip text="Job" term_id="job" >}}，
-可能以后会扩展以处理将完成执行的其他资源，例如 Pod 和自定义资源。
+TTL-after-finished {{<glossary_tooltip text="controller" term_id="controller">}} 提供了一种 TTL 机制来限制已完成执行的资源对象的生命周期。
```

Comment:
```suggestion
TTL-after-finished {{<glossary_tooltip text="控制器" term_id="controller">}} 提供了一种
TTL 机制来限制已完成执行的资源对象的生命周期。
```

Reply:
Made some adjustments as suggested，thanks for your review